### ISSUE TITLE
TreeGrid: Expand/Collapse buttons are not accessible using the keyboard

### ISSUE THREAD

**Issue by jmartirez** [infor-design/enterprise#5217]
**Describe the bug**
The actionable elements on the table are currently not accessible using the keyboard. The tabindex="-1" attribute is present on all the active elements within the table causing keyboard users to not be able to tab to it though it is actionable.

**To Reproduce**
on your local enterprise-ng

    1. run npm start
    2. Go to http://localhost:4000/components/datagrid/test-tree-multiselect.html
    3. navigate to the expand/collapse(+/-) buttons on the soho tree grid via keyboard, you cannot access the buttons via tab.

**Expected behavior**
[VPAT Recommendations]

    1. Developers must ensure that the tabindex="-1" is removed from actionable items that need to be reached via the keyboard, in addition to the mouse.
    2. Additionally, developers must ensure that the equivalent keyboard event handlers are also present for these elements.

**Version**

  - ids-enterprise-ng: v4.51.1

**Platform**

  - Device (if applicable) :NA
  - OS Version: Windows 10
  - Browser Name: Google Chrome
  - Browser Version:  90.0.4430.212 

Additional context
[https://jira.lawson.com/browse/LMCLIENT-34746]

**Comment by tmcconechy** [comment]
If you focus the cell you can use the space key to expand. But i think we need aria-expanded or a role on the cell.
Currently we use arrow keys not tab to move between cells.

We could also focus the button instead of the cell but we need to keep using arrows. Might be what they want from the testers

**Comment by tmcconechy** [comment]
Actually on further review i think there isnt anything we can do about this. We do have this working with the keyboard but we do not want to force extra tabbing in the datagrid. So you can use the expand/collapse button as follows

- tab into the grid
- use arrow keys to move to the expand/collapse cell
- hit space bar to toggle the expand/collapse

Any reason that wont work?
